### ISSUE TITLE
[Question] docker run get more arguments

### ISSUE THREAD

**Issue by pherbel** [Microsoft/VSTS-Docker-Preview#13]
How Can I pass more arguments for the docker run command?
e.g: 
-t
 --cap-addd
--security-opt

**Comment by markgarcia** [comment]
I also have this need with VSTS's Docker Compose "Run Service Image" action. Essentially, I want to [do this](http://stackoverflow.com/a/20919894). However I can't simply invoke the command manually as looking at the build logs it looks like the VSTS agent is inserting additional docker-compose files.

**Comment by fredgate** [comment]
Yes this is a essential feature. When I run image like nanoserver/apache24, the container is immediatly stopped. I need to use the -t option to run command.
Else this task is useless.

**Comment by gjsduarte** [comment]
+1
